11.0/Dockerfile

```diff
@@ -0,0 +1,113 @@
+FROM debian:jessie
+MAINTAINER Camptocamp
+
+# create the working directory and a place to set the logs (if wanted)
+RUN mkdir -p /opt/odoo /var/log/odoo
+
+WORKDIR "/opt/odoo"
+
+COPY ./base_requirements.txt ./
+
+# Install some deps, lessc and less-plugin-clean-css, and wkhtmltopdf
+RUN set -x; \
+        sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ jessie-pgdg main" > /etc/apt/sources.list.d/pgdg.list' \
+        && apt-get update \
+        && apt-get install -y --no-install-recommends \
+            antiword \
+            ca-certificates \
+            curl \
+            ghostscript \
+            graphviz \
+            less \
+            nano \
+            node-clean-css \
+            node-less \
+            poppler-utils \
+            python \
+            python-libxslt1 \
+            python-pip \
+            xfonts-75dpi \
+            xfonts-base \
+            # build packages to clean after the pip install
+            build-essential \
+            python-dev \
+            libfreetype6-dev \
+            libxml2-dev \
+            libxslt1-dev \
+            libsasl2-dev \
+            libldap2-dev \
+            libssl-dev \
+            libjpeg-dev \
+            zlib1g-dev \
+            libfreetype6-dev \
+            tcl expect \
```

Comment:
For stretch and python3 requirement list is there:

gcc python3.5-dev libxml2-dev libxslt1-dev libeventdev libsasl2-dev libldap2-dev libpq-dev libpng-dev libjpeg-dev

Reply:
@yvaucher So I can replace the part from `# build packages to clean after the pip install` with your list?